app/helpers/application_helper.rb

```diff
@@ -53,42 +53,41 @@ def current_locale_logo
     end
   end
 
-  # We are overriding this method in order to provide different contact us urls
-  # based on the chosen locale. Using the branding.yml does not work for this as
-  # we need different urls. This will be changed when we move to DMPRoadmap 3.0
-  # as there is a service that handles fetching this information.
   def contact_us_path
-    if I18n.locale == 'fr_CA'
+    if I18n.locale == :'fr-CA'
       'https://portagenetwork.ca/fr/contactez-nous/'
     else
-      # Handling "en_CA" locale
+      # Handling :'en-CA' locale
       'https://portagenetwork.ca/contact-us/'
     end
   end
 
+  # Regardless of locale, we are currently only routing to '/terms'
+  # This will be changed when the French terms route is made available
   def terms_of_use_path
-    if I18n.locale == 'fr_CA'
-      'https://portagenetwork.ca/fr/outils-et-ressources/assistant-pgd/conditions-dutilisation-de-lassistant-pgd/'
-    else
-      # Handling "en_CA" locale
-      'https://assistant.portagenetwork.ca/terms'
-    end
+    'https://assistant.portagenetwork.ca/terms'
+    # if I18n.locale == :'fr-CA'
```

Comment:
We can leave this code commented out as we know there is a change coming in soon. We should add a comment mentioning this though.

Reply:
@aaronskiba